packages/jest-worker/src/workers/ChildProcessWorker.ts

```diff
@@ -45,10 +46,14 @@ export default class ChildProcessWorker implements WorkerInterface {
   private _onProcessEnd!: OnEnd;
   private _request: ChildMessage | null;
   private _retries!: number;
+  private _stderr: ReturnType<typeof mergeStream> | null;
```

Comment:
ugh, but they don't export the interface: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b32ee2597779e6376db8c49e68d75c68e50b9287/types/merge-stream/index.d.ts#L11-L14